components/Aduan/DetailAduan/index.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="flex flex-col">
+    <span
+      class="font-bold text-[16px] text-gray-900 dark:text-dark-text-high leading-5 font-roboto"
+    >Detail Aduan</span>
```

Comment:
### Discussion

Same here about the usage of the `span` tag. 

WDYT?

Reply:
i fix [here](https://github.com/jabardigitalservice/super-app-web/pull/63/commits/fa6235acb6320da5348940d2e6981ebada0b06fb)